workflows/resources_test_scripts/bd_rhapsody_1_10_1_wta_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# settings
+ID=bd_rhapsody_1_10_1_wta_test
```

Comment:
Can you rename this to `ID=bd_rhapsody_wta_1_10_1_test` or even `ID=bd_rhapsody_wta_1.10.1_test`?